### ISSUE TITLE
Karma reporting framework contains an non-functioning API (unable to write JSON result to file)

### ISSUE THREAD

**Issue by devanshah2** [IBMa/equal-access#6]
Currently there exists an issues open in the open source karma tool where we are not able to send object from client side to server side: https://github.com/karma-runner/karma/issues/2192

Currently they have no plan to fix the issues, until there are more votes on the open source karma issues.

Therefore I am implementing a way for people to still be able to write the results to JSON from karma so that they can generate baselines files for their scans.
